docs/api/lib-app.adoc

```diff
@@ -0,0 +1,441 @@
+= Application library
+:toc: right
+:imagesdir: ../images
+
+Functions to fetch and manipulate applications.
+
+== Usage
+
+Add the following into your `build.gradle` file:
+
+[source,groovy]
+----
+dependencies {
+  include "com.enonic.xp:lib-app:${xpVersion}"
+}
+----
+
+In your JavaScript controller, add a require statement:
+
+```js
+var appLib = require('/lib/xp/app');
+```
+
+You are now free to use the library functionality.
+
+== Functions
+
+=== get
+
+Returns installed application. If both local and virtual apps are installed with specified key, then local will be in the result.
+
+[.lead]
+Parameters
+
+[%header,cols="1%,1%,97%a"]
+[frame="none"]
+[grid="none"]
+|===
+| Name   | Type   | Description
+| params | object | JSON with parameters
+
+[%header,cols="1%,1%,97%a"]
+[frame="topbot"]
+[grid="none"]
+[caption=""]
+.Properties
+!===
+! Name  ! Type   !  Description
+! key  ! string !  Application key
+!===
+
+|===
+
+[.lead]
+Returns
+
+*object* : (<<#Application,`Application`>>) Detail information for the application. Or null if the application could not be found.
```

Comment:
Properties object of an installed application with the specified key, or `null` if the application could not be found.